docs/README.md

```diff
@@ -14,35 +14,89 @@ an issue:
-* [Testing on Headless CI Systems (Travis, Jenkins)](tutorial/testing-on-headless-ci.md)
+  * [Mac App Store](tutorial/mac-app-store-submission-guide.md)
+  * [Windows Store](tutorial/windows-store-guide.md)
+  * [Snapcraft](tutorial/snapcraft.md)
+  * [Supported Platforms](tutorial/supported-platforms.md)
+* [Application Security](tutorial/security.md)
+  * [Reporting Security Issues](tutorial/security.md#reporting-security-issues)
+  * [Chromium Security Issues and Upgrades](tutorial/security.md#chromium-security-issues-and-upgrades)
+  * [Electron Security Warnings](tutorial/security.md#electron-security-warnings)
+  * [Security Checklist](tutorial/security.md#checklist-security-recommendations)
+* [Application Updates](tutorial/updates.md)
+  * [Deploying an Update Server](tutorial/updates.md#deploying-an-update-server)
+  * [Implementing Updates in Your App](tutorial/updates.md#implementing-updates-in-your-app)
+  * [Applying Updates](tutorial/updates.md#applying-updates)
+* [Application Debugging](tutorial/application-debugging.md)
+  * [Debugging the Main Process](tutorial/debugging-main-process.md)
+  * [Using Selenium and WebDriver](tutorial/using-selenium-and-webdriver.md)
+  * [Testing on Headless CI Systems (Travis, Jenkins)](tutorial/testing-on-headless-ci.md)
+  * [DevTools Extension](tutorial/devtools-extension.md)
+
+## Detailed Tutorials
+
+These individual tutorial expand on topics discussed in the guide above.
+
+* [In Detail: Installing Electron](tutorial/installation.md)
+  * [Global versus Local Installation](tutorial/installation.md#global-versus-local-installation)
+  * [Proxies](tutorial/installation.md#proxies)
+  * [Custom Mirrors and Caches](tutorial/installation.md#custom-mirrors-and-caches)
+  * [Troubleshooting](tutorial/installation.md#troubleshooting)
+* [In Detail: Electron's Versioning Scheme](tutorial/electron-versioning.md)
+  * [semver](tutorial/electron-versioning.md#semver)
+  * [Stabilization Branches](tutorial/electron-versioning.md#stabilization-branches)
+  * [Beta Releases and Bug Fixes](tutorial/electron-versioning.md#beta-releases-and-bug-fixes)
+* [In Detail: Packaging App Source Code with asar](tutorial/application-packaging.md)
+  * [Generating asar Archives](tutorial/application-packaging.md#generating-asar-archives)
+  * [Using asar Archives](tutorial/application-packaging.md#using-asar-archives)
+  * [Limitations](tutorial/application-packaging.md#limitations-of-the-node-api)
+  * [Adding Unpacked Files to asar Archives](tutorial/application-packaging.md#adding-unpacked-files-to-asar-archives)
+* [In Detail: Using Pepper Flash Plugin](tutorial/using-pepper-flash-plugin.md)
+* [In Detail: Using Widevine CDM Plugin](tutorial/using-widevine-cdm-plugin.md)
 * [Offscreen Rendering](tutorial/offscreen-rendering.md)
-* [Keyboard Shortcuts](tutorial/keyboard-shortcuts.md)
-* [Updating Applications](tutorial/updates.md)
 
-## Tutorials
+____
```

Comment:
Wondering if we prefer `---` to `____` in the docs

Reply:
I'd vote for `---` too. We use that on the website to denote the "summary" cut-off point for blog posts. It's a carryover convention from Jekyll.